src/components/app/DowntimeWarning.js

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+import React, { PureComponent } from 'react';
+
+import { Warning } from '../shared/Warning';
+
+export class DowntimeWarning extends PureComponent<{||}> {
+  _onDowntimeActionClick() {
+    window.open('/docs/#/./cloud-migration', '_blank');
+  }
+
+  render() {
+    // Because this code is planned to be only temporary, let's skip it in tests.
+    if (process.env.NODE_ENV === 'test') {
+      return null;
+    }
```

Comment:
Without this, a lot of tests need to be changed. I thought that for a temporary warning it wasn't worth it. But tell me what you think!